tests/integration/telemetry/stackdriver/testdata/client_request_count.json.tmpl

```diff
@@ -25,7 +25,8 @@
           "source_owner": "kubernetes://apis/apps/v1/namespaces/{{ .EchoNamespace }}/deployments/clt-{{ .ClusterName }}-v1",
           "source_principal": "spiffe://{{ .TrustDomain }}/ns/{{ .EchoNamespace }}/sa/default",
           "source_workload_name": "clt-{{ .ClusterName }}-v1",
-          "source_workload_namespace": "{{ .EchoNamespace }}"
+          "source_workload_namespace": "{{ .EchoNamespace }}",
+          "proxy_version": "unknown"
```

Comment:
Do you know why it is unknown? Are we not sending ISTIO_VERSION anymore?

Reply:
Fixed.